package.json

```diff
@@ -83,7 +84,7 @@
     "jsdoc": "^3.6.4",
     "lint-staged": "^10.1.2",
     "prettier": "^2.0.4",
-    "react-axe": "^3.4.1",
+    "react-axe": "3.3.0",
```

Comment:
See: https://github.com/dequelabs/react-axe/issues/74#issuecomment-599754902

It looks like many folks are experiencing similar issues with `react-axe@3.4.1`. In `3.3.0`, the error disappears.